validation/dam_break_2d/validation_dam_break_2d.jl

```diff
@@ -0,0 +1,158 @@
+
+include("../validation_util.jl")
+
+using TrixiParticles
+using JSON
+
+gravity = 9.81
+
+H = 0.6
+W = 2 * H
+tspan = (0.0, 8.0 / sqrt(gravity / H))
+
+# `particle_spacing` in this case is set relative to `H`, the initial height of the fluid.
+# Use H / 80, H / 320 for validation.
+# Note: H / 320 takes a few hours!
+particle_spacing = H / 40
+smoothing_length = 3.5 * particle_spacing
+smoothing_kernel = WendlandC2Kernel{2}()
+
+fluid_density = 1000.0
+sound_speed = 20 * sqrt(gravity * H)
```

Comment:
As discussed in the previous PR, this is not necessary when you just `trixi_include` the WCSPH simulation before the EDAC simulation.

Reply:
Yes I did not want to discuss that there.
So what I have done before is the following if I just wanted to update one of the reference files:
1) Comment out one of the trixi_includes and run the validation setup with the resolution.
2) Commit the reference file. 
3) Wonder later what happened when this actually fails when running the validation.

So what happened?
I have run the validation setup in a REPL session after running another example, which did use a different value which was not defined... And this took me hours to figure that out since it only happened with the highest resolution reference file... 
So I am really not sure this crap is worth it to remove two lines.

Reply:
Good point! Please keep it, then.